addons/website/tests/test_performance.py

```diff
@@ -106,7 +106,7 @@ def test_15_perf_sql_queries_page(self):
 
     def test_20_perf_sql_queries_homepage(self):
         # homepage "/" has its own controller
-        self.assertEqual(self._get_url_hot_query('/'), 13)
+        self.assertEqual(self._get_url_hot_query('/'), 14)
```

Comment:
Probably @Gorash didn't yet processed https://github.com/odoo/odoo/pull/88276#discussion_r856059547
There shouldn't be any read on the website table, there probably still is.
I didn't checked the branch in local to have an in depth look at those queries, I'll try to have a look to see what's wrong

Reply:
The cache of the whole page (and therefore no cfr token...) has been removed in favor of the t-cache. There is a request in the attributes of the <html> tag. I did not make a system to cache values (t-set-) and therefore the request is made (on a master data since it is the company).